graspologic/utils/__init__.py

```diff
@@ -45,5 +46,6 @@
     "fit_plug_in_variance_estimator",
     "remove_vertices",
     "remap_labels",
+    "remap_node_ids",
     "to_weighted_edge_list",
```

Comment:
This is missing in the import list... didn't dig into it but I suspect there is an issue.

Reply:
`to_weighted_edge_list` to be clear

Reply:
to_weighted_edge_list should have been removed in a PR that merged to dev a while ago.  when did you cut this branch?

Reply:
augh I missed the string representation in `__all__`: https://github.com/microsoft/graspologic/blob/dev/graspologic/utils/__init__.py#L48

Can you delete it in this PR?  it shouldn't be there.

Reply:
Sure